File: android/src/main/java/io/boxo/expo/LottieProgressOptions.kt
package io.boxo.expo

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

class LottieProgressOptions : Record {
    @Field
    val light: String = ""

    @Field
    val dark: String? = null

    @Field
    val width: Int? = null

    @Field
    val height: Int? = null

}

File: ios/LottieProgressOptions.swift
//
//  LottieProgressOptions.swift
//  ExpoBoxoSdk
//

import ExpoModulesCore

struct LottieProgressOptions: Record {
    @Field
    var light: String = ""

    @Field
    var dark: String?

    @Field
    var width: Int?

    @Field
    var height: Int?
}
